### ISSUE TITLE
Inform user a restart is required for update settings, or refresh settings after Save

### ISSUE THREAD

**Issue by HowardWolosky** [Code52/carnac#267]
I just started trying out Carnac today and tried to change the fadeout delay, but it didn't work.
I searched through the issues/PR's and came across [this comment](https://github.com/Code52/carnac/pull/133#pullrequestreview-32974622) from #133:

> Just one minor thing, changing the value requires a restart of the app, but I guess there's not much we can do about that as the settings are read at startup?

From an end-user perspective, I just figured the feature was broken.

I see three potential ways to improve the experience for the end-user:

1. Add a note at the top of the settings page that says something to the effect: "Any changes made will require a restart of the app to take effect."
2. When clicking `Save`, have a pop-up of some sort say: "A restart of the app is required for these changes to take effect."
  2b. In that pop-up have a button that offers to restart the app for the user (a la VS Code after an update).
3. Refactor the settings logic so that it can be updated dynamically after the user hits `Save` so that it just works.

**Comment by AaronLLTT** [comment]
This should definitely be done. The third option is the best, but probably the most time consuming. Option 2 is vital, as I was just trying this app and about to dismiss it as garbage until I found this post.

**Comment by vishnuvardhan-s** [comment]
I agree on point 2. I also thought it was a bug or some missing deps and later saw this [comment](https://github.com/Code52/carnac/pull/133#pullrequestreview-32974622) by searching through Issues/PRs.
